lib/rust/config-reader/src/lib.rs

```diff
@@ -93,12 +95,14 @@ pub fn generate_config_module_from_yaml(config_path: impl AsRef<std::path::Path>
             for (key, value) in mapping {
                 let key = key.as_str().unwrap().to_snake_case();
                 let value = value.as_str().unwrap();
-                def.push_str(&format!("{}pub {}: &'static str,\n", indent, key));
-                inst.push_str(&format!("{}{}: \"{}\",\n", indent, key, value));
-                vars.push_str(&format!(
-                    "#[allow(non_upper_case_globals)]\npub const {}: &str = \"{}\";\n",
+                writeln!(def, "{}pub {}: &'static str,", indent, key).unwrap();
+                writeln!(inst, "{}{}: \"{}\",", indent, key, value).unwrap();
+                writeln!(
+                    vars,
+                    "#[allow(non_upper_case_globals)]\npub const {}: &str = \"{}\";",
                     key, value
-                ));
+                )
+                .unwrap();
             },
```

Comment:
I don't like these changes TBH. They are using `unwrap` now and no unwrap in the previous version. Why these changes?